api/v1alpha1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1alpha1 contains API Schema definitions for the misf v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=misf.ibm.com
+// +kubebuilder:object:generate=true
+// +groupName=misf.ibm.com
```

Comment:
- afaik, build markers shouldn't have space b/n marker & comment

Reply:
Maybe, but every time i save a file in my editor the default golang beautifier kicks in and adds a space

Reply:
Same for me. lets have this change